File: src/Page/ApprovedCoursesPage.java
package Page;

import System.FunctionType;
import System.SystemMessage;

public class ApprovedCoursesPage extends Page{

	public ApprovedCoursesPage(String content) {
		super(content);
		setType(PageType.APPROVED_COURSES_PAGE);
		setName("Approved Courses Page");
	}
	
	@Override
	public SystemMessage runPage() {
		showContent();
		takeInput();
		return new SystemMessage(FunctionType.CHANGE_PAGE, PageType.MAIN_MENU_PAGE, null);
	}
}

File: iter3/python_development/Page/ApprovedCoursesPage.py
from iter3.python_development.Page.Page import Page
from iter3.python_development.Page.PageType import PageType
from iter3.python_development.System.FunctionType import FunctionType
from iter3.python_development.System.SystemMessage import SystemMessage

class ApprovedCoursesPage(Page):

    def __init__(self, content):
        super().__init__(content)
        self.setType("APPROVED_COURSES_PAGE")
        self.setName("Approved Courses Page")

    
    def runPage(self):
        self.showContent()
        self.takeInput()
        return SystemMessage("CHANGE_PAGE", "MAIN_MENU_PAGE", None)